modules/viewing-pipeline-artifacts.adoc

```diff
@@ -0,0 +1,32 @@
+:_module-type: PROCEDURE
+
+[id="viewing-pipeline-artifacts_{context}"]
+= Viewing pipeline artifacts
+
+[role='_abstract']
+When a pipeline run executes, you can view its pipeline artifacts from the {productname-short} dashboard. Pipeline artifacts can help you evaluate the performance of your pipeline runs and make it easier to understand your pipeline components. Pipeline artifacts can range from plain text data or detailed, interactive data visualizations.  
+
+.Prerequisites
+* You have logged in to {productname-long}.
+ifndef::upstream[]
+* If you are using specialized {productname-short} groups, you are part of the user group or admin group (for example, {oai-user-group} or {oai-admin-group}) in OpenShift.
+endif::[]
+ifdef::upstream[]
+* If you are using specialized {productname-short} groups, you are part of the user group or admin group (for example, {odh-user-group} or {odh-admin-group}) in OpenShift.
+endif::[]
+* You have previously created a data science project that is available and contains a pipeline server.
+* You have imported a pipeline to an active pipeline server.
+* You have previously triggered a pipeline run.
+
+.Procedure
+. From the {productname-short} dashboard, click *Experiments* -> *Artifacts*.
+. Select the *project* that contains the pipeline experiment for the pipeline artifacts that you want to view.
+. Study the table showing a list of pipeline artifacts.
```

Comment:
This isn't really a UI step - I'd suggest removing it, and if we really need something here, this would be an appropriate place for a validation step. (e.g. 'A list of pipeline artifacts is displayed.')

Reply:
Removing this as there is already a verification step of the same nature.